src/core/radio/radio.hpp

```diff
@@ -537,6 +537,15 @@ class Radio : public InstanceLocator, private NonCopyable
      *
      */
     Error EnableCsl(uint32_t aCslPeriod, otShortAddress aShortAddr, const otExtAddress *aExtAddr);
+
+    /** Resets CSL receiver parameters in radio to default values.
+     *
+     * @retval  kErrorNotImplemented Radio driver doesn't support CSL.
+     * @retval  kErrorFailed         Other platform specific errors.
+     * @retval  kErrorNone           Successfully enabled or disabled CSL.
+     *
+     */
+    Error ResetCslParameters(void);
```

Comment:
```suggestion
    Error ResetCsl(void);
```
I think we can just name the method as `ResetCsl` which corresponds to `EnableCsl`.

Reply:
Done